src/tools/ResultsComparer/Program.cs

```diff
@@ -185,6 +187,54 @@ private static void ExportToCsv((string id, Benchmark baseResult, Benchmark diff
             Console.WriteLine($"CSV results exported to {csvPath.FullName}");
         }
 
+        private static void ExportToXml((string id, Benchmark baseResult, Benchmark diffResult, EquivalenceTestConclusion conclusion)[] notSame, FileInfo xmlPath)
+        {
+            if (xmlPath == null)
+            {  
+                Console.WriteLine("No file given");
+                return;
+            }
+
+             if (xmlPath.Exists)
+                xmlPath.Delete();
+
+            using (XmlWriter writer = XmlWriter.Create(xmlPath.Open(FileMode.OpenOrCreate, FileAccess.Write, FileShare.Write)))
+            {
+                writer.WriteStartElement("performance-tests");
+                foreach (var slower in notSame.Where(x => x.conclusion == EquivalenceTestConclusion.Slower))
+                {
+                    writer.WriteStartElement("test");
+                    writer.WriteAttributeString("name", slower.id);
```

Comment:
does `WriteAttributeString` takes care of escaping the string?

benchmark id contains the benchmark arguments which can be literally anything, including `<` and `>` characters. Example:

```cs
[Benchmark]
[Arguments(">>>>")]
public int Example(string arg) => arg.GetHashCode();
```

Reply:
It does, I just tested it.